### DISCUSSION TITLE
text with new line

### DISCUSSION THREAD

**Discussion by znaniye** [johnfercher/maroto#406] [Q&A]
with v2, how can I create dynamic text with new lines that will be passed as a function argument? What I tried to do, the text always expands horizontally infinitely and doesn't change to a new line.

**Comment by miguelbernadi** [comment]
I'm having the same issue. In my case, the text can wrap around the size of the column, but I need definite line breaks at parts to make the column multi-line.

**Comment by lordofscripts** [comment]
I need this too.
